app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -153,6 +154,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    userReadMessage: {
```

Comment:
Does 'hasUserReadMessage' more descriptive?

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/4224/commits/20d22f85305b8493d1c62277dedfbb7a1da2b613